### ISSUE TITLE
Hacktober Feature Request: Add Accessibility tests 

### ISSUE THREAD

**Issue by wordshaker** [dotnetnotts/dotnetnotts-web#31]
### Describe the feature

- Add pa11y to CI build to demonstrate how changes impact accessibility

### Advantages

- Helps the repo keep an accessibility focus

## Links

- [pa11y ](https://github.com/pa11y/pa11y-ci)

#### Acceptance Criteria

Functional and non-functional criteria that should be met by this work

- [ ] Pally runs on every PR build

**Comment by wordshaker** [comment]
Completed by @rach-m
